be2-scala/src/main/scala/ch/epfl/pop/config/application.conf

```diff
@@ -1,8 +1,7 @@
 ## Akka configs 
-akka.http.server.idle-timeout =  5 s  
-#akka.http.server.idle-timeout = infinite   # for testing purposes only
-akka.http.client.idle-timeout =  5 s 
 
+akka.http.client.idle-timeout =  600 s 
+akka.http.server.idle-timeout =  600 s  

```

Comment:
is there a reason for switching from 5s to 600s to fix the open roll-call bug?

Reply:
No the timeouts does not make any difference